sklearn/tests/test_metaestimators_metadata_routing.py

```diff
@@ -277,7 +284,10 @@ def enable_slep006():
     LassoLarsCV(),
     MultiTaskElasticNetCV(),
     MultiTaskLassoCV(),
+    OneVsOneClassifier(ConsumingClassifier()),
```

Comment:
Yep and we should not have the OneVsOne and other multiclass learner. I assume this is a remaining from a merge conflict. Good catch @OmarManzoor